guides/v2.1/extension-dev-guide/cli-cmds/cli-howto.md

```diff
@@ -35,20 +35,63 @@ Following is a summary of the process:
 1.	Create a Command class (the recommended location is `<your component root dir>/Console/Command`).
 
 	See [`<Magento_Store_module_dir>/Console/Command/StoreListCommand.php`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Store/Console/Command/StoreListCommand.php) for example.
+
+```php
+<?php
+    namespace Magento\CommandExample\Console\Command;
+    
+    use Symfony\Component\Console\Command\Command;
+    use Symfony\Component\Console\Input\InputInterface;
+    use Symfony\Component\Console\Input\InputOption;
+    use Symfony\Component\Console\Output\OutputInterface;
+    
+    /**
+     * Class SomeCommand
+     */
+    class SomeCommand extends Command
+    {
+        /**
+         * @inheritDoc
+         */
+        protected function configure()
+        {
+            $this->setName('my:first:command');
+            $this->setDescription('This is my first console command.');
+            
+            parent::configure();
+        }
+    
+        /**
+         * @param InputInterface $input
+         * @param OutputInterface $output
+         *
+         * @return void
```

Comment:
Please, change the return type to `int|null` according to the parent class.

Reply:
Thank you, the return type has been adjusted.